tests/e2e-utils/README.md

```diff
@@ -0,0 +1,35 @@
+# WooCommerce End to End Test Utilities 
```

Comment:
We probably need to add more information here about basic/helper flows like setup, product creates, etc, also about how do we add initial data (after we decide it). Not a blocker for this PR though.

Reply:
Agreed

Reply:
Going to address this in a different PR.